### DISCUSSION TITLE
Cannot Export Changes to Branch

### DISCUSSION THREAD

**Discussion by untxtledstudent** [community/community#101662] [Codespaces]
### Select Topic Area

Bug

### Body

Whenever I try to export my codespaces to branch because I dont have any more allowed time it loads for a long time and then it sends an error

![Screenshot 2024-01-23 8 33 36 AM](https://github.com/community/community/assets/156121894/5bc0df0e-02cb-450c-ab10-bfd0aac1afff)

**Comment by RuBeeN-cmd** [comment]
Same Problem

**Comment by AmraniCh** [comment]
same problem

**Comment by jameszokah** [comment]
facing the same issue here please, help!

**Reply by jameszokah** [reply]
@osortega please help
